src/Satooshi/Bundle/CoverallsV1Bundle/Entity/Exception/RequirementsNotSatisfiedException.php

```diff
@@ -24,6 +24,34 @@ class RequirementsNotSatisfiedException extends \RuntimeException
     protected $readEnv;
 
     /**
+     * Array of secret env vars.
+     *
+     * @staticvar array
```

Comment:
this tag isn't allowed too...

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md